src/ros_numpy/registry.py

```diff
@@ -32,6 +32,21 @@ def numpify(msg, *args, **kwargs):
 			raise ValueError("Cannot determine the type of an empty Collection")
 		conv = _to_numpy.get((msg[0].__class__, True))
```

Comment:
need to hit this path too.

I suggest moving your checks to `normalize_rosbag_cls`, then you can just drop in here and on line 29:

```
cls = normalize_rosbag_cls(msg[0].__class__)
```